Vpn.Service/ManagerConfig.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Coder.Desktop.Vpn.Service;
+
+[SuppressMessage("ReSharper", "AutoPropertyCanBeMadeGetOnly.Global")]
+public class ManagerConfig
+{
+    [Required]
+    [RegularExpression(@"^([a-zA-Z0-9_-]+\.)*[a-zA-Z0-9_-]+$")]
+    public string ServiceRpcPipeName { get; set; } = "Coder.Desktop.Vpn";
+
+    // TODO: pick a better default path
```

Comment:
Probably just AppData?

Reply:
Can't be writable by users so it will definitely be in ProgramData or Program Files